Bottom: f0126ca417d381c0e7f0413bdb1b43d1d1cfe1b3
Top:    b897fc99a4242c88e5ade20c46ad36b562632185
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-21 15:13:13 +0000

More work in progress


---

diff --git a/src/polygon1.c b/src/polygon1.c
index c6b2ffc..42d94af 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -3076,8 +3076,6 @@ M_Set_Parentage (POLYAREA *poly, POLYPARENTAGE parentage)
 
 }
 
-static void clear_marks (POLYAREA * p);
-
 /* just like poly_Boolean but frees the input polys */
 int
 poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
